### PR TITLE
Fixing bug 2320 and 2321 - Hide KC connectors' stacktraces into a spoiler #2320 #2321 

### PR DESCRIPTION
**Pull request by Kris-K-Dev**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

- Do not display "null" as a stacktrace in KC #2321 
- Hide KC connectors' stacktraces into a spoiler #2320

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [X] Manually (please, describe, if necessary)
- [X] Unit checks
- [ ] Integration checks
- [X] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**
Screenshot 2022-08-05 at 23 41 43

### CODE REVIEW

File: kafka-ui-react-app/src/components/Connect/Details/Tasks/Tasks.tsx [hunk 1]
```diff
@@ -45,7 +47,9 @@ const Tasks: React.FC = () => {
             <td>
               <Tag color={getTagColor(task.status)}>{task.status.state}</Tag>
             </td>
-            <td>{task.status.trace || 'null'}</td>
+            <td>
+              <TraceCell stackTrace={task.status.trace || ''} />
+            </td>
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/Details/Tasks/Tasks.tsx:52]: I would suggest to use TopicMessages approach. So we need to add expandable row with message

Comment (KriKiparoidze) [reply kafka-ui-react-app/src/components/Connect/Details/Tasks/Tasks.tsx]: Fixed

File: kafka-ui-react-app/src/components/Connect/Details/Tasks/TraceContent/TraceContent.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import * as S from './TraceContent.styled';
+
+export interface TraceContentProps {
+  traceContent?: string;
+}
+
+const TraceContent: React.FC<TraceContentProps> = ({ traceContent }) => {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/Details/Tasks/TraceContent/TraceContent.tsx:9]: there is no need in a property. Pls use children

File: kafka-ui-react-app/src/components/Connect/Details/Tasks/TraceContent/__test__/TraceContent.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      <table>
+        <tbody>
+          <TraceContent traceContent={traceContent} />
+        </tbody>
+      </table>
+    );
+
+  it('renders Trace Content', () => {
+    const { getByText } = setupComponent('Lorem ipsum');
+    expect(getByText('Lorem ipsum')).toBeInTheDocument();
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/Details/Tasks/TraceContent/__test__/TraceContent.spec.tsx:17]:
```suggestion
    renderCompoent('Lorem ipsum');
    expect(screen.getByText('Lorem ipsum')).toBeInTheDocument();
```

File: kafka-ui-react-app/src/components/Connect/Details/Tasks/TraceContent/__test__/TraceContent.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from 'lib/testHelpers';
+import TraceContent from 'components/Connect/Details/Tasks/TraceContent/TraceContent';
+
+describe('Trace Content', () => {
+  const setupComponent = (traceContent: string) =>
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/Details/Tasks/TraceContent/__test__/TraceContent.spec.tsx:6]:
```suggestion
  const renderComponent = (traceContent: string) =>
```

**Pull request closed without merging**